java-api-examples/run-inverse-text-normalization-transducer.sh

```
@@ -19,6 +19,6 @@ if [ ! -f ./itn_zh_number.fst ]; then
 fi
 
 java \
-  -Djava.library.path=$PWD/../build/lib \
+  -Dsherpa_onnx.native.path=$PWD/../build/lib \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)